zipkin-api.yaml

```diff
@@ -139,12 +140,24 @@ paths:
         required: true
         description: the 64-bit hex-encoded id of the trace as a path parameter.
         type: string
+      - name: raw
```

Comment:
thanks! I was mucking around with the old one.. seems result is no longer
defined when JSON is ok. Will rebase on yours.